messages/teleporter/message_manager.go

```diff
@@ -70,6 +74,24 @@ func NewMessageManager(
 	}, nil
 }
 
+func isDestination(messageDestinationID ids.ID, allowedDestinationID ids.ID) bool {
```

Comment:
dont see us using this, can you also change it to `isDestinationChain`

Reply:
just saw Michael's comment, agree we can remove the check, can we remove this helper function?

Reply:
Removed